app/models/concerns/acts_as_sortable.rb

```diff
@@ -0,0 +1,44 @@
+module ActsAsSortable
```

Comment:
Added this module to share the `update_numbers!` method across multiple models

Reply:
Thanks, it's good to get behavior like this generic!